bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -685,7 +685,7 @@ public Image(Device device, ImageFileNameProvider imageFileNameProvider) {
 	initialNativeZoom = DPIUtil.getNativeDeviceZoom();
 	ElementAtZoom<String> fileName = DPIUtil.validateAndGetImagePathAtZoom (imageFileNameProvider, getZoom());
 	if (fileName.zoom() == getZoom()) {
-		long handle = initNative (fileName.element(), getZoom());
+		long handle = initNative (fileName.element(), getZoom(), SWT.IMAGE_COPY);
```

Comment:
Hmm, we're not really wanting to copy something here. I would e.g. use SWT.DEFAULT (-1) here and change the default case in the switch in  copyImageDataForDisabledOrGrayImages like
```
default: {
	returnImageData = data;
	break;
}
```
I'm not 100% sure about that, but using SWT.IMAGE_COPY (just from the semantics) sounds not correct either.

Reply:
You are right, but I wanted to stay away from default. It could give the users a wrong impression if they use some wrong flag. It's just my opinion.